src/controllers/controller.doughnut.js

```diff
@@ -76,7 +76,12 @@ module.exports = function(Chart) {
 
 				for (i = 0, ilen = (chart.data.datasets || []).length; i < ilen; ++i) {
 					meta = chart.getDatasetMeta(i);
-					meta.data[index].hidden = !meta.data[index].hidden;
+					try {
```

Comment:
Probably not, I'll continue to investigate if there is a better way to fix this issue.